src/extension.ts

```diff
@@ -1198,3 +1204,114 @@ function toggleBooleanWorkspaceSetting(setting: string) {
   const currentValues = configProperty?.workspaceValue ?? false;
   config.update(setting, !currentValues, ConfigurationTarget.Workspace);
 }
+
+class SymbolItem implements QuickPickItem {
```

Comment:
can we move all this custom symbol search stuff into another file? It would easier to remove if/when the issue is fixed upstream

Reply:
Done!